hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/connector/jdbc/JdbcSqlConnector.java

```diff
@@ -378,7 +380,7 @@ public Vertex deleteProcessor(@Nonnull DAG dag, @Nonnull Table table0) {
      */
     @SuppressWarnings("ReturnCount")
     private QueryDataType resolveType(String columnTypeName) {
-        switch (columnTypeName.toUpperCase()) {
+        switch (StringUtil.upperCaseInternal(columnTypeName)) {
```

Comment:
nit: you can static import this method for more compact code

Reply:
Used static import